app/src/main/res/xml/app_widget_info.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<appwidget-provider xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Is it possible to add widget images like this? we can give more insight to the users about the widget.

![Screenshot_20220103-212835](https://user-images.githubusercontent.com/22434597/147977167-b18f6216-501d-4b04-b736-df4fee6dd377.png)

Reply:
I completely agree with you. I thought about it but appwidget-provider is just accept xml or icon. Because I created the widget with Glance, it is not possible with Jetpack Compose but I can create xml version if it is necessary for us.

Reply:
Do images come from the `xml`? I didn't know that, so let's open a new task for it, maybe we can add it in the future.